### DISCUSSION TITLE
今年有一起看Ti的吗

### DISCUSSION THREAD

**Discussion by Eurus1997** [share121/inter-knot#2625] [其他]
十年刀斯林了，现在玩不动了，变成云玩家了。但是Ti还是要看的，你知道的，我是Ame十年老粉。
![20240801115130](https://github.com/user-attachments/assets/257fd1ac-3567-4cdf-bd9d-5dc93b65be47)

**Comment by FRIKY-c** [comment]
如果不是星铁赞助我还以为刀塔在中国已经寄了
